core/src/main/java/org/elasticsearch/index/query/MissingQueryBuilder.java

```diff
@@ -30,18 +40,26 @@
 
     public static final String NAME = "missing";
 
-    private String name;
+    public static final boolean DEFAULT_NULL_VALUE = false;
 
-    private Boolean nullValue;
+    public static final boolean DEFAULT_EXISTENCE_VALUE = true;
 
-    private Boolean existence;
+    private String fieldPattern;
+
+    private Boolean nullValue = DEFAULT_NULL_VALUE;
+
+    private Boolean existence = DEFAULT_EXISTENCE_VALUE;
 
     static final MissingQueryBuilder PROTOTYPE = new MissingQueryBuilder(null);
 
-    public MissingQueryBuilder(String name) {
-        this.name = name;
+    public MissingQueryBuilder(String fieldPattern) {
+        this.fieldPattern = fieldPattern;
```

Comment:
maybe I'm missing something, but the comment is referring to the builder, and we don't check for null there. Another example is CommonTermsQuery which performs a check in the Parser and in validate but not in the constructor method as well.